cwms-data-api/src/test/resources/cwms/cda/data/dto/timeseriesprofile/timeseriesprofile.json

```diff
@@ -0,0 +1,8 @@
+{
+  "description": "Description",
+  "parameter-list": ["Temperature","Depth"],
+  "location-id": "Location",
```

Comment:
would want more input but for the Project DTO we went with an embedded location object with a minimum of office,location-id, but could include everything

Reply:
Agreed, this should be re-using the location definition that is being done in related tickets. Also, the Java classes that are generating this JSON should be added to this PR.

In the database the parameters have a position, what is the reason for dropping that here?